extensions/amp-bind/amp-bind.md

```diff
@@ -89,30 +98,25 @@ In this example, `myState.foo` will evaluate to `"bar"`.
 - For security reasons, binding to `innerHTML` is disallowed
 - All attribute bindings are sanitized for unsafe URL protocols (e.g. `javascript:`)
 
-### Bindable attributes
+#### Element-specific attributes
 
-For non-AMP elements, most attributes accepted by the [AMP Validator](https://validator.ampproject.org/) are bindable.
+Most attributes accepted by the [AMP Validator](https://validator.ampproject.org/), for AMP and non-AMP elements, are bindable.
 
-For AMP components, the height and width attributes are bindable along with the following specific attributes:
+There are a few special bindable attributes without non-bindable siblings:
```

Comment:
This sentence confuses me.  So, does the table below show elements that are bindable but only for those attributes? Also, you say a "few" but there's two.  Can you clarify this area.  Thx